### PR TITLE
fix: updated react dependency in podspec to enable build in Xcode 12 (for iOS >= 12)

### PR DESCRIPTION
**Pull request by alexisbronchart**
See https://github.com/facebook/react-native/issues/29633#issuecomment-694187116

### CODE REVIEW

**Review by karvulf — APPROVED** [review]

**Comment by chrisbobbe** [comment]
Just a heads-up 🙂: this would break compatibility with RN v0.59 and earlier because `React-Core` doesn't exist as its own  pod until https://github.com/facebook/react-native/commit/2321b3fd7, which was released in RN v0.60.

The range for `react-native` under `peerDependencies` in package.json currently (as of 4.0.1; [link](https://github.com/react-native-cameraroll/react-native-cameraroll/blob/v4.0.1/package.json#L42)) contains versions that would no longer work; I would recommend narrowing that range so that it excludes them, and announcing this as a breaking change in a major-version release.

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
   "license": "MIT",
   "peerDependencies": {
     "react": "^16.0",
-    "react-native": ">=0.57 <=0.62"
+    "react-native": ">=0.60 <=0.63"
```
Comment (chrisbobbe) [inline package.json:42]: I don't think the `React` to `React-Core` change lets us raise this upper bound to 0.63; I think that would best be done as part of #233 (I've just opened #264 for that).

Comment (alexisbronchart) [reply package.json]: Allright it's back to 0.62

**Review by Naturalclar — APPROVED** [review]
@alexisbronchart @chrisbobbe Thanks!

**Pull request merged**